Utilities/macrecovery/macrecovery.py

```diff
@@ -204,6 +204,10 @@ def save_image(url, sess, filename='', directory=''):
         'Cookie': '='.join(['AssetToken', sess])
     }
 
+    if not os.path.exists('com.apple.recovery.boot'):
+        os.mkdir('com.apple.recovery.boot')
+    directory = 'com.apple.recovery.boot'
```

Comment:
Declare this variable first and then reuse it?

Reply:
Ohh, you mean declaring once `com.apple.recovery.boot` as a `str` variable and then reuse it?

Reply:
Yes.

Reply:
Can you please check latest commit?